File: src/Aircraft.Web/web/src/app/models/stage.ts
export class Stage {
    description : string;
    createdAt : Date;
    status : boolean;
    type: number;
    value: string;
    maintenanceId: string;

    constructor(description : string, createdAt : Date,  type: number, value : string, status : boolean, maintenanceId: string) {
        this.description = description;
        this.createdAt = createdAt;
        this.type = type;
        this.value = value;
        this.status = status;
        this.maintenanceId = maintenanceId;
    }

}

File: src/Aircraft.Domain/Stage.cs
using System;

namespace Aircraft.Domain
{
    public class Stage
    {
        public Guid Id { get; set; }
        public Maintenance Maintenance { get; set; }
        public Guid MaintenanceId { get; set; }
        public string Description { get; set; }
        public int Type { get; set; }
        public bool Status { get; set; }
        public string Value { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
    }
}
